README.md

```diff
@@ -6,20 +6,29 @@ A set of tools for developing [Fathom](http://mozilla.github.io/fathom/) ruleset
 
 Here is an example of how you might use FathomFox to develop a Fathom ruleset that finds the semi-transparent overlays behind in-page pop-ups:
 
-1. Fork, clone, and install the [Fathon-Trainees](https://github.com/mozilla/fathom-trainees/) repository as per its instructions.
-2. Install dependencies and build with `yarn build`, then run `yarn browser` to launch Firefox. (`yarn install` (or just `yarn`) will fail with an error about an incompatible node version, so be sure to use `yarn build` to install modules, or pass the `--ignore-engines` switch to `yarn install`.)
-3. Using `about:debugging`, install your fathom-trainees extension by selecting its `addon/manifest.json` file.
-4. Install [Fathom-Fox](https://addons.mozilla.org/firefox/addon/fathomfox/).
-5. Navigate to a web page that has what you're trying to recognize: a background overlay behind a modal dialog.
-6. Using Firefox's developer tools inspector, select the overlay element.
-7. Switch to the Fathom developer tools tab, enter "overlay" in the label field, and click Save Page.
-8. After you have labeled about 20 pages like this, click the FathomFox icon in the main toolbar, and choose Vectorizer.
```

Comment:
> I tried to bulk freeze the following pages: https://npr.org, https://nytimes.com

I'm afraid those work for me! I do get some CSP errors in the console, but they don't seem to stop it working. (Attaching this to an arbitrary line so we can have a threaded discussion about it)

Reply:
Hm, yep I just did the following STR for Release 71 and Nightly 73, and it worked in both channels.

My default browser is Nightly, so I thought perhaps that was the difference, but no. The only other difference I can think of is that I deleted my `node_modules` folder prior to the instructions in the README this time. I will chalk this up to working too near the holidays, I suppose, heh.

**In Release 71**
1. `rm -rf node_modules/`
2. Follow instructions in "Get FathomFox Running" in the README
  * `yarn run build`
  *  Replace `yarn run browser` with `web-ext run -s addon/ -f firefox`, since my default browser is Nightly.
3. Follow instructions in the "Corpus Collector" section in the README
  * In the textarea in the Corpus Collector page, enter:
```
https://npr.org
https://nytimes.com
```

**In Nightly 73**
1. Repeat above but just use `yarn run browser`.